src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/SpotlightSubmissionController.java

```diff
@@ -106,6 +107,7 @@ public ResponseEntity<GetSpotlightSubmissionDataBySchemeIdDto> getSpotlightSubmi
     }
 
     @GetMapping(value = "/scheme/{schemeId}/download", produces = EXPORT_CONTENT_TYPE)
```

Comment:
Yeah most likely as I assume this will be triggered by an admin and not lambda. I'll add and then we can test the spotlight journeys when deployed to sandbox